### PR TITLE
JWWEB-736 New Transaction Item

### PR DESCRIPTION
**Pull request by Creepypoke**
What is done:
* Component are connected to redux store by `connect` already, and get information about transaction by txHash from it
* Showing transaction title/notes according local data (favorites, named addresses)
* Showing transaction status (fail, success, pending, stuck) according with [documentation](https://jibrelnetwork.atlassian.net/wiki/spaces/JWALLET/pages/769425458/Ethereum+blockchain+transactions)
* Showing formatted asset amount according asset

### CODE REVIEW

File: src/components/TransactionItemNew/transactionsIndex.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      return isPending && (timestampDiff > HALF_HOUR)
+    },
+  },
+]
+
+function getTransactionType(
+  state: AppState,
+  { from }: TransactionWithPrimaryKeys,
+): TransactionDirection {
+  const ownerAddress = selectActiveWalletAddress(state) || ''
```
Comment (IViol) [inline src/components/TransactionItemNew/transactionsIndex.js]: better to get it outside

File: src/components/TransactionItemNew/transactionsIndex.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+  return !!from && (ownerAddress.toLowerCase() === from.toLowerCase()) ? 'out' : 'in'
+}
+
+function getTransactionStatus(
+  state: AppState,
```
Comment (IViol) [inline src/components/TransactionItemNew/transactionsIndex.js]: state is not used

File: src/components/TransactionItemNew/transactionsIndex.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    // TODO: We can't get correct fiat amount yet, because we are have't exchange rate history
+    fiatAmount: '',
+  }
+}
+
+// FIXME: Temporary solution to store indexed values for transactions
+export function transactionsIndex(state: AppState) {
+  const transactions = selectTransactionsList(state)
+
+  return transactions.reduce((prev, transaction) => ({
```
Comment (IViol) [inline src/components/TransactionItemNew/transactionsIndex.js]: reduceResult

File: src/components/TransactionItemNew/TransactionItem.js [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+class TransactionItem extends PureComponent<Props, *> {
+  static defaultProps = {
+    onClick: noop,
+  }
+
+  handleClick = (event: SyntheticEvent<HTMLAnchorElement>): void => {
+    event.preventDefault()
+
+    if (!this.props.isActive) {
+      // $FlowFixMe
```
Comment (IViol) [inline src/components/TransactionItemNew/TransactionItem.js]: ?

Comment (Creepypoke) [reply src/components/TransactionItemNew/TransactionItem.js]: Flow don't understand that `onClick?: (Address) => mixed` property with default value `noop` from `lodash` always is function.

**Pull request merged**